.circleci/config.yml

```diff
@@ -140,10 +140,10 @@ jobs:
             fi
 
   publish_pypi:
-    <<: *container_python27
+    <<: *container_python36
     steps:
       - checkout
-      - *restore_cache_python27
+      - *restore_cache_python36
       - run:
           name: Deploy to PyPi
           command: |
```

Comment:
this command still refers to `venv27`?

Reply:
(L150 onwards, couldn't comment on it directly)

Reply:
Yes, indeed, that would fail.   Good catch.   893034b